Makefile

```diff
@@ -0,0 +1,30 @@
+SOURCE_FILES?=$$(glide novendor)
+TEST_PATTERN?=.
+TEST_OPTIONS?=
+
+setup: ## Install all the build and lint dependencies
+	curl https://glide.sh/get | sh
+	go get -u github.com/kisielk/errcheck
+	go get -u github.com/golang/lint/golint
+	glide install
+
+test: ## Run all the tests
+	go test $(TEST_OPTIONS) -cover $(SOURCE_FILES) -run $(TEST_PATTERN) -timeout=30s
+
+lint: ## Run all the linters
+	errcheck $(SOURCE_FILES)
+	go vet $(SOURCE_FILES)
+	golint ./... | grep -v vendor
+
+ci: lint test ## Run all the tests and code checks
+
+build: ## Build a beta version of releaser
+	go build
```

Comment:
Right now I agree it doesn't look very helpful, my experience is that over time `go build` is not going to be enough to actually build the project so I always have such a task. It acts as docs too in a way as, long term, it will  explain how to build the project.

One good example is adding custom building IDs to binaries.